presidio-analyzer/presidio_analyzer/predefined_recognizers/medical_license_recognizer.py

```diff
@@ -21,8 +21,9 @@ class MedicalLicenseRecognizer(PatternRecognizer):
     PATTERNS = [
         Pattern(
             "USA DEA Certificate Number (weak)",
-            r"[a-zA-Z]{2}\d{7}|[a-zA-Z]{1}9\d{7}",
```

Comment:
Are the two new regex in addition or instead this weak one? Could you please put a link to the source in a comment on this file?

Reply:
The one from the other comment (https://www.meditec.com/blog/dea-numbers-what-do-they-mean) is good